app/config/config.yml

```diff
@@ -224,12 +231,12 @@ old_sound_rabbit_mq:
         mailjet_campaign:
             connection:       default
             exchange_options: { name: 'mailjet-delayed-campaign', type: direct }
-            class:            AppBundle\Producer\MailjetProducer
+            class:            AppBundle\Producer\MailerProducer
```

Comment:
Can you please base this PR on the other branch to have a lighter diff? I'm stopping here the review for now.

Reply:
Done